apps/ecosystem/src/ecosystem.repository.ts

```diff
@@ -1369,4 +1369,134 @@ export class EcosystemRepository {
       throw error;
     }
   }
-}
+
+  async getEcosystemsByOrgId(orgId: string): Promise<IEcosystemData[]> {
+    try {
+
+      const ecosystemsDetails = await this.prisma.ecosystem.findMany({
+        where: {
+          ecosystemOrgs: {
+            some: {
+              orgId
+            }
+          }
+          
+        },
+        include: {
+          ecosystemOrgs: {
+            include: {
+              ecosystemRole: true
+            }
+          }
+        }
+      });
+      
+      return ecosystemsDetails;
+    } catch (error) {
+      this.logger.error(`Error in getting ecosystems: ${error.message}`);
+      throw error;
+    }
+  }
+
+  async getOrgName(orgId: string): Promise<{
+    name: string;
+  }> {
+    try {
+      const orgName = await this.prisma.organisation.findUnique({
+        where: {
+          id: orgId
+        },
+        select: {
+          name: true
+        }
+      });
+
+      return orgName;
+    } catch (error) {
+      this.logger.error(`Error in getting organization names: ${error.message}`);
+      throw error;
+    }
+  }
+
+  async getEcosystemLeadOrgDetails(orgId: string): Promise<{
+    name: string;
+  }> {
+    try {
+      const orgName = await this.prisma.organisation.findUnique({
```

Comment:
We are directly calling the organization table from the ecosystem MS

Reply:
Resolved